### PR TITLE
add arm64 builds in the ct-ng latest

### PR DESCRIPTION
**Pull request by grooverdan**
rebased version of #129 

extends previous #130

### CODE REVIEW

**Comment by grooverdan** [comment]
Yes, #130 has the same commits, merging that will make the common commits here removed (or just merge this and close the rest). I've also included most of @dbroemmel #129 is here (except admin-cross-builds.sh)

File: gcc-cross/Dockerfile [hunk 1]
```diff
@@ -4,6 +4,9 @@ MAINTAINER Matt Godbolt <matt@godbolt.org>
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN mkdir -p /opt mkdir -p /home/gcc-user && useradd gcc-user && chown gcc-user /opt /home/gcc-user
+
+RUN apt-get clean -y && apt-get check -y
```
Comment (mattgodbolt) [inline gcc-cross/Dockerfile]: Ditto my comment in the other PR review

File: gcc-cross/Dockerfile [hunk 2]
```diff
@@ -44,6 +53,15 @@ RUN curl -sL http://crosstool-ng.org/download/crosstool-ng/crosstool-ng-1.23.0.t
     ./configure --enable-local && \
     make -j$(nproc)
 
+RUN curl -sL https://github.com/crosstool-ng/crosstool-ng/archive/master.zip --output crosstool-ng-master.zip  && \
```
Comment (mattgodbolt) [inline gcc-cross/Dockerfile]: Ditto my comment in the other PR

Comment (grooverdan) [reply gcc-cross/Dockerfile]: Added bf8bb0e

**Comment by mattgodbolt** [comment]
Thanks! I'm going to merge these in and see where we are at the end!

**Pull request merged**